site/search/SearchChartHitComponent.tsx

```diff
@@ -40,21 +32,9 @@ const SearchChartHitLarge = (
 ): React.ReactElement => {
     const isMediumScreen = useMediaQuery(MEDIUM_BREAKPOINT_MEDIA_QUERY)
 
-    // If the hit doesn't support rich data display, render the fallback component
-    if (!hasRichDataDisplay(props.hit))
-        return <SearchChartHitRichDataFallback {...props} />
-
     // On smaller screens, render the medium variant which implements the
     // mobile layout for smaller viewports
     if (isMediumScreen) return <SearchChartHitMedium {...props} />
 
     return <SearchChartHitRichData variant="large" {...props} />
 }
-
-function hasRichDataDisplay(hit: SearchChartHit): boolean {
-    // CSV-based explorers don't support rich data display
-    return !(
-        hit.type === ChartRecordType.ExplorerView &&
-        hit.explorerType === ExplorerType.Csv
-    )
-}
```

Comment:
Since the data table is now built on the server, CSV-based explorers automatically support rich data display too